crates/runner/src/vm/instructions/store.rs

```diff
@@ -0,0 +1,446 @@
+//! STORE instructions for the Cairo M VM.
+//!
+//! STORE instructions are used to store values in the memory.
+//! They are used to store the result of an operation or the value of a variable.
+
+use crate::{
+    memory::{Memory, MemoryError},
+    vm::{instructions::Instruction, State},
+};
+
+/// Store the sum of the values at the offsets `fp + off0` and `fp + off1` in the memory at the offset `fp + off2`.
+///
+/// OPCODE ID: 0
```

Comment:
I am not sure this should be in comments.
This is subject to change depending on optimisation.